ibm_catalog.json

```diff
@@ -50,41 +58,67 @@
             ],
             "configuration" : [
               {
-                "key": "ibmcloud_api_key"
-              },
-              {
-                "key": "use_existing_resource_group"
+                "key": "ibmcloud_api_key",
+                "required": true,
+                "display_name": "IBM Cloud API Key",
+                "description": "IBM Cloud API Key to use for provisioning resources"
               },
               {
-                "key": "resource_group_name",
-                "required": true
+                "key": "existing_resource_group_name",
+                "display_name": "Resource group",
+                "default_value": "dev",
```

Comment:
I do not think we want a default for this one as its not likely dev will be an existing RG